typescript/nomad-deploy/package.json

```diff
@@ -28,12 +28,14 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@ethersproject/experimental": "^5.3.0",
+    "@gnosis.pm/safe-ethers-adapters": "file:../../../safe-core-sdk/packages/safe-ethers-adapters",
```

Comment:
why?

Reply:
dont know, probably just an artifact. removed it

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/f100d7a7785c3405fce74cb9bd7fdfd1a8f8287f